src/pmacfiltercontrol/pmacFilterControlWrapper.py

```diff
@@ -232,6 +232,27 @@ def __init__(
             "ATTENUATION", *ATTENUATION, on_update=self._set_manual_attenuation
         )
 
+        self.hist_high3 = builder.aOut(
+            "HIST:HIGH3",
+            on_update=lambda val: self._set_hist("High3", val),
+        )
+        self.hist_high2 = builder.aOut(
+            "HIST:HIGH2",
+            on_update=lambda val: self._set_hist("High2", val),
+        )
+        self.hist_high1 = builder.aOut(
+            "HIST:HIGH1",
+            on_update=lambda val: self._set_hist("High1", val),
+        )
+        self.hist_low2 = builder.aOut(
+            "HIST:LOW2",
+            on_update=lambda val: self._set_hist("Low2", val),
+        )
+        self.hist_low1 = builder.aOut(
+            "HIST:LOW1",
+            on_update=lambda val: self._set_hist("Low1", val),
+        )
+
```

Comment:
I was thinking you could store these in a dictionary at this point, rather than as class members and then constructing the dictionary in `_setup_hist_thresholds`. You could also do that in a loop over `("High3", "High2", ...)`  to make it less verbose.

Reply:
Is commit bc70a5d what you meant by this?

Reply:
Yep